README.md

```diff
@@ -90,11 +103,13 @@ All available console fonts can be found in `/usr/share/kbd/consolefonts` and se
 
 # Scripting your own installation
 
+> **To create your own ISO with this script in it:** Follow [ArchISO](https://wiki.archlinux.org/index.php/archiso)'s guide on creating your own ISO.
```

Comment:
Why is this quoted?

Reply:
I see this was just moved from below, this should probably not be a quoted text in the original version either...